src/porepy/geometry/intersections.py

```diff
@@ -405,8 +419,8 @@ def polygons_3d(polys, target_poly=None, tol=1e-8, include_point_contact=True):
             segment, False if vertex. The index identifies the vertex, or the first
             vertex of the segment. If the intersection is in the interior of a polygon,
             the tuple is replaced by an empty list.
-        list of list of bool: For each polygon, for all intersection points, True if this
-            intersection is formed by a single point.
+        np.ndarray of list of bool: For each polygon, for all intersection points,
```

Comment:
As an array, with items of type object. It is not a beautiful construct, I prefer lists, and would like to do a purge of arrays that should be lists throughout the code, but in many cases (like this one) changing it will have consequences up- and downstream. Not this PR.